iconservice/icon_constant.py

```diff
@@ -59,10 +59,10 @@
 class ConfigKey:
     BUILTIN_SCORE_OWNER = 'builtinScoreOwner'
     SERVICE = 'service'
-    SERVICE_FEE = 'fee'
-    SERVICE_AUDIT = 'audit'
-    SERVICE_DEPLOYER_WHITELIST = 'deployerWhiteList'
-    SERVICE_SCORE_PACKAGE_VALIDATOR = 'scorePackageValidator'
+    SERVICE_FEE = 'FEE'
+    SERVICE_AUDIT = 'AUDIT'
+    SERVICE_DEPLOYER_WHITELIST = 'DEPLOYER_WHITELIST'
+    SERVICE_SCORE_PACKAGE_VALIDATOR = 'SCORE_PACKAGE_VALIDATOR'
```

Comment:
Those 'value' parts should not be changed to uppercase since those are 'key' of JSON strings.